ui/cryptomaterial/dropdown.go

```diff
@@ -189,14 +211,19 @@ func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
 		layout.Rigid(func(gtx C) D {
 			gtx.Constraints.Max.X = dropdownItemWidth - gtx.Dp(values.MarginPadding50) // give some space for the dropdown Icon
 			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			if item.DisplayFn != nil {
+				return item.DisplayFn(gtx)
+			}
+
 			return layout.Inset{
 				Right: unit.Dp(5),
 				Left:  unit.Dp(5),
 			}.Layout(gtx, func(gtx C) D {
 				lbl := d.theme.Body2(item.Text)
-				if !d.isOpen && len(item.Text) > 14 {
-					lbl.Text = item.Text[:14] + "..."
+				if !d.isOpen && len(item.Text) > 20 {
+					lbl.Text = item.Text[:20-3] + "..."
 				}
```

Comment:
Why 20?

Reply:
There’s enough space even with the default dropdown width.

Reply:
On a bigger window, won't 20 or even 25 still leave some space? Is it possible to actually detect how many letters can actually fit into the available space? PS: Out of scope for this PR, but leave a TODO. And also define this value as a constant somewhere rather than as a magic number.

Reply:
> On a bigger window, won't 20 or even 25 still leave some space? 

The display uses a constant width.